### PR TITLE
#163371781 Update REST API endpoint to fetch a specific meetup from the database

### PR DESCRIPTION
**Pull request by khwilo**
#### What does this PR do?

Fetches data of a specific meetup from the database table

#### Description of Task to be completed?

- [x] Update the implementation for getting one meetup from a Python list to a database table
- [x] Have the endpoint `GET /api/v2/meetups/<meetup-id>` working

#### How should this be manually tested?

1. Clone the repository and cd into it.
2. Create a virtual environment  and activate it:
```bash
$ python3 -m venv venv
$ source venv/bin/activate
```
3. Install the required dependencies:
```bash
$ pip install -r requirements.txt
```

4. Create a Postgres database for development and test environment

5. View the env-sample file on the environment variables to export

6. Perform the database migrations:
```bash
$ python3 manage.py
```
7. Run the Flask application
```bash
$ flask run
```
8. On Postman test the endpoint with the following header:
    - *key*: **Content-Type**
    - *value*: **application/json**

#### What are the relevant pivotal tracker stories?

[#163371781](https://www.pivotaltracker.com/story/show/163371781)

#### Screenshot

v2-fetch-one-meetup

### CODE REVIEW

File: app/api/v2/models/meetup_model.py [hunk 1]
```diff
@@ -31,15 +31,29 @@ def save(self):
         self.cursor.execute(query)
         self.connection.commit()
 
+    def get_meetup_by_id(self, meetup_id, value):
+        '''Return a meetup given a meetup id'''
```
Comment (munyili-samuel) [inline app/api/v2/models/meetup_model.py:35]: Please use double quotes (`"`) for DocStrings - PEP8

File: app/tests/v2/sample_data.py [hunk 2]
```diff
@@ -86,10 +86,10 @@
 
 NEW_MEETUP = dict(
     location="Test New Location",
-    images=[],
+    images='{}',
     topic="Test New Topic",
     happeningOn="Mar 5 2019 11:00AM",
-    tags=["Programming", "Design"]
+    tags='{Programming, Design}'
```
Comment (munyili-samuel) [inline app/tests/v2/sample_data.py:92]: Just thinking, tags should be a list ([...]) and not Objects or Dictionaries ({...}). Any reason why opted to use the object?

**Review by munyili-samuel — COMMENTED** [review]
Nice work @khwilo,
Please check out my comments

File: app/api/v2/models/meetup_model.py (hunk 1 above)
Comment (khwilo) [reply app/api/v2/models/meetup_model.py:35]: Alright, I will update in next PR.

File: app/tests/v2/sample_data.py (hunk 2 above)
Comment (khwilo) [reply app/tests/v2/sample_data.py:92]: Please refer to this PR #22. I have used the Postgresql syntax for array of strings. When I used the Python list of syntax, I was encountering a malformed syntax error.

**Pull request merged**